pipeline/ffmpeg.go

```diff
@@ -129,15 +129,13 @@ func (f *ffmpeg) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 	sourceSegments := sourceManifest.GetAllSegments()
 	job.sourceSegments = len(sourceSegments)
 
-	if job.sourceSegments > 0 && job.InputFileInfo.Format != "hls" {
-		firstSeg := sourceSegments[0]
-		lastSeg := sourceSegments[job.sourceSegments-1]
+	if job.sourceSegments > 1 && job.InputFileInfo.Format != "hls" {
+		segmentsToCheck := []int{0, 1, job.sourceSegments - 2, job.sourceSegments - 1}
```

Comment:
hmm this is still a workaround right? Did we ever narrow down the root cause? Was it a bug in ffmpeg segmenting?

Reply:
Seems to be an ffmpeg but yeah, will be raising a ticket.

Reply:
what's the bug specifically? what types of sources result in this bug?

Reply:
OK i'll add this to my backlog of video bugs to investigate. Seems alright for now.